plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/gmodel/AbstractGModelCreateNodeOperationHandler.java

```diff
@@ -59,7 +60,10 @@ public void executeOperation(final CreateNodeOperation operation) {
          .map(location -> LayoutUtil.getRelativeLocation(location, container));
       GModelElement element = createNode(relativeLocation, operation.getArgs());
       container.getChildren().add(element);
-      actionDispatcher.dispatchAfterNextUpdate(new SelectAction(), new SelectAction(List.of(element.getId())));
+      actionDispatcher.dispatchAfterNextUpdate(
```

Comment:
Having to manually add the Subclient id to actions that should be dispatched after the next update is not ideal.
Wouldn't it be better to rework the `dispatchAfterNextUpdate` mechanism in the action dispatcher directly?
We should store rework the postUpdateQueue to a Map<string,List<Action>>. Whenever the `dispatchAfternextUpdate` method is invoked we then store the action in the map entry for the corresponding subclient id of the action.
When we clear the postUpdateQueue (`dispatchPostUpdateQueue`) we should only remove the actions that have the same subclientid as the corresponding updateModelAction.